test/driver.js

```diff
@@ -582,6 +584,18 @@ var Driver = (function DriverClosure() { // eslint-disable-line no-unused-vars
       this._log('Done !');
       this.end.textContent = 'Tests finished. Close this window!';
 
+      var re = new XMLHttpRequest();
+      re.open('POST', '/browserTestReports', false);
+      re.setRequestHeader('Content-Type', 'application/json');
+      re.responseType = 'arraybuffer';
```

Comment:
The response is not used anywhere, so there is no need to specify `responseType`. Remove this line.

Reply:
ok